central/notifiers/acscsemail/client_impl.go

```diff
@@ -48,7 +49,7 @@ func ClientSingleton() Client {
 	return client
```

Comment:
Cannot comment on l.45 but found this browsing through: we typically avoid reusing the `http.DefaultClient` and specifcially create a client like `client := &http.Client{Transport: proxy.RoundTripper()}`.

While the default HTTP client respects proxy settings via environment variables, there's also a configuration file that cna be used to configure SOCKS proxies and others. While we may control things within ACSCS ourselves in terms of configuration, I'd just be consistent and use it here to avoid running into trouble down the road of missing proxy support (like we did when we initially had sensor running behind a proxy with ACSCS).

Reply:
Awesome, good catch. I've changed it like suggested 👍